src/plot_api/edit_types.js

```diff
@@ -0,0 +1,84 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var isPlainObject = require('../lib/is_plain_object');
+
+module.exports = {
```

Comment:
good idea :tada: 

I don't think this deserved a file of its own though. Putting this in

https://github.com/plotly/plotly.js/blob/master/src/plot_api/helpers.js

should suffice.